dp/drivers/ixgbe.c

```diff
@@ -765,6 +764,7 @@ static struct ix_eth_dev_ops eth_dev_ops = {
 	.promiscuous_disable = generic_promiscuous_disable,
 	.reta_update = reta_update,
 	.rx_queue_setup = rx_queue_setup,
+	.rx_idle_wait = rx_idle_wait,
```

Comment:
I would reference this function like the `poll` function here: https://github.com/aroulin/ix/blob/master/dp/drivers/ixgbe.c#L485